src/meta/assessment/originalDataPoint/odps/updateNationalClass.ts

```diff
@@ -40,7 +40,7 @@ const calculateValues = (nationalClass: ODPNationalClass) => {
     otherPlantedForestPercent: getValueOrNull(otherPlantedForestPercent, rowIsMaxedForestCharacteristics),
     forestPercent: getValueOrNull(forestPercent, rowIsMaxedExtentOfForest),
     otherWoodedLandPercent: getValueOrNull(otherWoodedLandPercent, rowIsMaxedExtentOfForest),
-    forestPlantationIntroducedPercent: forestPlantationIntroducedPercent ?? '0',
+    forestPlantationIntroducedPercent,
```

Comment:
no, it shouldn't. I think that getValueOrNull is used to check if a in-row sum is 100% or not. the case of forestPlantation is different because we have only one value per row.